src/models/v2/asyncapi.ts

```diff
@@ -25,4 +52,40 @@ export class AsyncAPIDocument extends Mixin(BaseModel, ExtensionsMixin) implemen
       )
     );
   }
+
+  channels(): ChannelsInterface {
+    return new Channels(
+      Object.entries(this._json.channels || {}).map(([channelAddress, channel]) => 
+        this.createModel(Channel, channel, { id: channelAddress, address: channelAddress, pointer: `/channels/${tilde(channelAddress)}` })
+      )
+    );
+  }
+
+  operations(): OperationsInterface {
+    const operations: OperationInterface[] = [];
+    this.channels().forEach(channel => operations.push(...channel.operations().all()));
+    return new Operations(operations);
+  }
+
+  messages(): MessagesInterface {
+    const messages: MessageInterface[] = [];
+    this.operations().forEach(operation => messages.push(...operation.messages().all()));
+    return new Messages(messages);
+  }
+
+  schemas(): SchemasInterface {
+    return new Schemas([]);
+  }
```

Comment:
Atm we don't have implemented `traverse` function, so now function returns empty collections, it should be implemented and tested in next PRs.